OWNERS_ALIASES

```diff
@@ -0,0 +1,21 @@
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+aliases:
+  sig-apps-leads:
+    - michelleN
+    - mattfarina
+    - prydonius
+    - kow3ns
+  helm-admins:
+    - sgoings
```

Comment:
Please remove. As awesome as @sgoings is, he is not a core maintainer and therefore should not be pinged on issues/PRs. Let's just remove admin rights from non-maintainers instead. :)

Reply:
@bacongobbler should he then be removed from the @kubernetes/helm-admins team? the way this repo's collaborators are setup, that team currently grants him admin rights to this repo